generators/app/templates/src/constants/predicates.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Object containing a reference to the UI ontology's IRIs
+ */
+export const LDPVocabulary = {
```

Comment:
For the components library I'm using the following convention for vocabs:

```js
/* vocabs.js */
const lpdBase = `http://www.w3.org/ns/ldp#`
export const LPD = {
    BASIC_CONTAINER: `${lpdBase}BasicContainer`,
};
```
```js
/* client.js */
import { LPD } from '@vocabs';
...
```

It's up to you if you want to use the same idea here until we use LIT.